drivers/volume/portworx/portworx.go

```diff
@@ -2125,11 +2125,16 @@ func (p *portworx) CreatePair(pair *storkapi.ClusterPair) (string, error) {
 		return "", fmt.Errorf("cannot get cluster manager, err: %s", err.Error())
 	}
 
+	var credID string
+	if bkpl, ok := pair.Spec.Options[storkapi.BackupLocationResourceName]; ok {
```

Comment:
Add a version check for 2.6.0 here and return an error if it is an older version